src/services/ClaudyActionComponents/Support.jsx

```diff
@@ -22,9 +22,11 @@ export class Support extends Component {
       nextProps.opened ? this.onOpen() : this.onReturn()
     }
 
-    // reset message if successfully sent
     if (nextProps.emailStatus.isSent && this.props.emailStatus.isSending) {
+      // reset message if successfully sent
```

Comment:
I feel that the code is enough self-explainatory to avoid those two comments. What do you think ?

Reply:
Hmm… Agree, finally the line is more to explain the `successfully sent` rather than the `reset message` 😆